docusaurus/docs/learn/core-concepts/security/authentication/10-third-party-cas.md

```diff
@@ -79,89 +76,55 @@ The [Edge Management API](/docs/reference/developer/api#edge-management-api) acc
-which are stored in [SVIDs](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-verifiable-identity-document-svid).
+### OTT CA Enrollment
 
-3rd Party CAs support defining a set of x509 claims configuration that allows a claim to be matched to an identity
-`externalId`. The configuration is contained in an object in the field `externalIdClaims`. When not defined, x509
-client certificate authentication attempts to find an identity that is tied to an [authenticator](./auth#authenticators) 
-by matching client certificates. Using x509 claims, the client is matched by the identity `externalId` value.
+OTT CA Enrollment requires that the enrolling client also has an existing client certificate signed by a verified 3rd Party CA. When creating an identity the `id` of the target 3rd Party must be specified. This is not yet supported by the Ziti CLI.
 
-The fields under `externalIdClaims` is as follows:
+#### Create an Identity for a 3rd Party CA with Edge Management API
 
-- `location` - defines which value(s) in an x509 certificate will be processed: `COMMON_NAME`, `SAN_URI`, `SAN_EMAIL`
-- `matcher` - defines how values from `location` will be filtered: `ALL`, `PREFIX`, `SUFFIX`, `SCHEME`
-- `matcherCriteria` - defines the `PREFIX`, `SUFFIX`, or `SCHEME` to look for based on `matcher`
-- `parser` - defines how values from `location` filtered by `matcher` will be parsed: `NONE`, `SPLIT`
-- `parserCriteria` - defines the criteria to provide to `parser`
-- `index` - should multiple values still be available after `location`, `matcher,` and `parser` processing the integer value here will be used from the set
+`POST /edge/management/v1/identities`
 
-#### CA Create/Update REST API
 ```json
 {
-  "name": "myCA",
-  "certPem": "—–BEGIN CERTIFICATE—–\nMIIDdTCCAHMU...\n—–END CERTIFICATE—–",
-  "externalIdClaims": {
-    "location": "SAN_URI",
-    "matcher": "SCHEME",
-    "parser": "NONE",
-    "parserCriteria": "",
-    "index": 0
+  "name": "test-user10",
+  "type": "User",
+  "isAdmin": false,
+  "roleAttributes": [
+    "dial"
+  ],
+  "enrollment": {
+    "ottca": "<ott-ca-id>"
   }
 }
 ```
-#### Ziti CLI
-
-```
-ziti edge create ca myCa ca.pem -l SAN_URI -m SCHEME -x spiffe -p "NONE"
-```
-
-```
-ziti edge update ca myCa -l SAN_URI -m SCHEME -x spiffe -p "NONE"
-```
-
-### Location, Matcher, Parser
-
-x509 claims are located, matched, and parsed. Location defines where the value(s) are sourced from, matching filters, 
-and parsing allows for a single value to yield multiple claims.
-
-#### Location
-
-Location configuration sources value(s) from the x509 certificate
 
-- `COMMON_NAME` - the common name of the certificate
-- `SAN_URI` - SAN URI fields
-- `SAN_EMAIL` - SAN email fields
+The enrollment JWT can be retrieved by fetching the created identity by ID and parsing out the `enrollment.ottca.jwt` field. 
```

Comment:
This is planned for removal. The way to get it will be to request `GET /identities/:id/enrollments`.  Which lists all outstanding enrollments for a specific identity.

Reply:
That explains why I was struggling with `{{baseUrl}}/enrollments?filter=identityId%3D%22LcTkRcyVdM%22` not filtering enrollments by identity `identityId`.

Reply:
https://github.com/openziti/ziti-doc/pull/592/commits/7d2febee2161190ed08209c5c7d9750219aa9f91